README.md

```diff
@@ -1 +1,24 @@
-# hw01
\ No newline at end of file
+# About Me
+Hello, my name is Tianyi Qian.  
+
+### Educational Background  
+
+---
+
+I'm a grad student in the MAPSS program at the University of Chicago and I concentrate in **Quantitative Methods and Social Anlaysis**. I studied **Sociology** and **Media Studies** as an undergrad. I'm hoping to learn the following from this class:
```

Comment:
Merit: Nice use of bold here to emphasize your current concentration and undergrad major study fields! 

Improvement: A little suggestion would be that it could probably be clearer if you use bold and italics to illustrate different points, like using bold for your concentration and italics for your undergrad study areas.